drivers/bt_ll_nrfxlib/flash/flash_ll_nrfxlib.c

```diff
@@ -17,9 +17,10 @@
 #include "ble_controller_soc.h"
 #include "multithreading_lock.h"
 
-#define SOC_NV_FLASH_NODE DT_INST(0, soc_nv_flash)
+#define SOC_NV_FLASH_NODE DT_NODELABEL(flash0)
```

Comment:
what the reason for this change?

Reply:
why thid is not done like in zephyr/drivers/flash/soc_flash_nrf.c

Reply:
Upstream only does it that way because the upstream devicetree maintainer doesn't like node labels. I think it's kind of silly and verbose so I suggested doing it this way instead. All our SoCs already declare the node labels we need to keep these C files from having a bunch of ifdefs per-SoC.

Reply:
expressive comment :)
now understod